packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -59,6 +65,9 @@ test.describe('Prop: `isFullWidthBelowMid`', () => {
                 } as ModalProps,
             });
 
+            // Follow up to remove in Jan
+            await page.waitForTimeout(5000);
```

Comment:
Out of interest does `await page.waitForSelector(selector);` work?

Reply:
That does work however, as we learnt this morning, we've seen that you can't use a selector like `data-test-id` or something of the sort as it sits inside the shadow dom. Meaning we can't use waitForSelector to work. We were able to assert of the text on the label and confirmed this. 😢

Reply:
Ah i see. Thanks!